packages/itwinui-react/src/core/utils/components/Icon.tsx

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import '@itwin/itwinui-css/css/utils.css';
+
+export type IconProps = {
+  /**
+   * Size of the svg.
+   *  - 'auto': scales with text
+   *  - 'medium': works well with most text
+   *  - 'small': smaller than medium
+   *  - 'large': larger than medium
+   *
+   * @default 'auto'
+   */
+  size?: 'auto' | 'small' | 'medium' | 'large';
+  /**
+   * Fill (color) of the svg.
+   * Defaults to `--iui-color-icon-muted`. Can be specified to
+   * use one of the status colors.
+   *
+   * @default 'default'
+   */
+  fill?: 'default' | 'positive' | 'informational' | 'negative' | 'warning';
+} & React.ComponentProps<'span'>;
+
+const sizeMap = {
+  auto: 'auto',
+  small: 's',
+  medium: 'm',
+  large: 'l',
+} as const;
+
+/**
+ * A utility to component to provide size and fill to svgs.
+ * Works well with svgs from `@itwin/itwinui-icons-react`.
+ *
+ * @example
+ * <Icon size='default'>
+ *   <SvgPlaceholder />
+ * </Icon>
+ *
+ * @example
+ * <Icon fill='negative'>
+ *   <SvgStatusError />
+ * </Icon>
+ */
+export const Icon = (props: IconProps) => {
```

Comment:
I believe our users would benefit from having storybook examples of new Icon component. If we don't want to add it separately, then we should add usage examples to other components.

Reply:
added
https://itwin.github.io/iTwinUI/854/react/?path=/story/utilities-icon